### PR TITLE
[WIP] Particle drifts

### PR DESCRIPTION
**Pull request by StanczakDominik**
I'm making another attempt at getting through Bellan's fundamentals of plasma physics again and I figured I'd add a couple of the better known particle drifts.

This includes changes from #481 as I started this on top of those changes, so it should not be merged before that one.

I'm still figuring out a good way to include the grad B drift... perhaps that should simply be another argument.

### CODE REVIEW

File: plasmapy/physics/parameters.py [hunk 1]
```diff
@@ -778,11 +776,15 @@ def gyrofrequency(B, particle='e-', signed=False, Z=None):
...
-                       'T_i': {'units': u.K, 'can_be_nan': True},
-                       })
-def gyroradius(B, particle='e-', *, Vperp=np.nan * u.m / u.s, T_i=np.nan * u.K):
+@utils.check_quantity(B={'units': u.T},
+                      Vperp={'units': u.m / u.s, 'can_be_nan': True},
+                      T_i={'units': u.K, 'can_be_nan': True},
+                      )
+def gyroradius(B: u.T,
+               particle='e-',
+               *,
```
Comment (ritiek) [inline plasmapy/physics/parameters.py]: Not a review comment or anything but I am curious what `*` does here!

Comment (StanczakDominik) [reply plasmapy/physics/parameters.py]: It's a design decision I'm still queasy about, but we sort of . By adding *, you make sure the following arguments can only be input as keyword arguments - thus, one cannot mistake them.

It's weird, I know.

File: plasmapy/physics/drifts.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    """
+
+    # np.cross drops units right now, thus this hack
+    cross = np.cross(E.si.value, B.si.value) * E.unit * B.unit
+    return (cross / (B*B).sum(-1)).to(u.m/u.s)
+
+
+@utils.check_quantity()
+def force_drift(F: u.N, B: u.T, q: u.C):
```
Comment (namurphy) [inline plasmapy/physics/drifts.py]: We could add a return annotation here:  ` -> u.m/u.s:`

**Comment by StanczakDominik** [comment]
Gah. By the way, in the interest of *moving faster and* (hopefully not) *breaking* (too many) *things* (such as with pep8 and documentation fixes), I'm turning off this option in the repo settings:

> Dismiss stale pull request approvals when new commits are pushed
> New reviewable commits pushed to a matching branch will dismiss pull request review approvals.

**Comment by StanczakDominik** [comment]
Of course, thanks for the review! :)

**Pull request merged**